packages/perseus/src/widgets/__testdata__/interactive-graph.testdata.ts

```diff
@@ -2089,13 +2089,6 @@ export const segmentWithAllLockedRayVariations: PerseusRenderer = {
     },
 };
 
-export const segmentWithLockedFigures: PerseusRenderer =
-    interactiveGraphQuestionBuilder()
-        .addLockedPointAt(-7, -7)
-        .addLockedLine([-7, -5], [2, -3])
-        .addLockedEllipse([0, 5], [4, 2], {angle: Math.PI / 4})
-        .build();
-
```

Comment:
moved this because I wanted it after all the other locked figure testdata blocks